### ISSUE TITLE
how should we updated nix?

### ISSUE THREAD

**Issue by Fuseteam** [DavHau/nix-on-armv7l#5]
i took a quick look at the dockerfile and iiuc this is build from 19.09 should it be updated to build from 20.09?

**Comment by DavHau** [comment]
Yes, I think it would be great to update it.
Feel free to open a PR.

A big plus would be if it would be built by a github action.

**Comment by Fuseteam** [comment]
yeah was thinking about that, but my attempts to build armv7 binaries with github action aren't exactly successful so far

**Comment by Fuseteam** [comment]
i attempted to build via a github action anyway, so far I'm running into errors: https://github.com/Fuseteam/nix-on-armv7l/runs/2169469019?check_suite_focus=true

**Comment by DavHau** [comment]
This whole project will hopefully become obsolete after https://github.com/NixOS/nix/pull/2667
It might not be worth putting any more effort into it.
The mentioned PR just needed to add binaryTarball targets as well, which currently seem to be missing.

**Comment by Fuseteam** [comment]
oh cool so official support is in the work, awesome!

**Comment by Fuseteam** [comment]
hmmm you haven't received an response

**Comment by Fuseteam** [comment]
looks like it is merged! but i have no idea how to make use of it lol
